src/tools/error-codes/main.rs

```diff
@@ -0,0 +1,57 @@
+use std::env::args;
+use std::fs::read_dir;
+use std::path::Path;
+use std::process::{exit, Command};
+
+fn get_error_codes<P: AsRef<Path>>(folder_path: P) -> Vec<String> {
+    let mut ret = Vec::with_capacity(10);
+
+    for entry in read_dir(folder_path.as_ref()).expect("read_dir failed") {
+        let entry = entry.expect("Couldn't unwrap entry");
+        let path = entry.path();
+
+        if !path.is_file() {
+            continue;
+        }
+        ret.push(format!("{}", path.display()));
```

Comment:
You never seem to use the String-ness of the error codes, so maybe you can just return `Vec<PathBuf>`? If we do use it let's call display() there and not here.

Reply:
I use it as an argument and print it afterwards. Both cases require strings.